src/sql/workbench/services/accountManagement/browser/accountDialog.ts

```diff
@@ -357,13 +362,20 @@ export class AccountDialog extends Modal {
 			this.vstelemetryService
 		);
 
-		Registry.as<IViewsRegistry>(ViewContainerExtensions.ViewsRegistry).registerViews([{
-			id: newProvider.addedProvider.id,
-			name: newProvider.addedProvider.displayName,
-			ctorDescriptor: new SyncDescriptor(AccountPanel),
-		}], ACCOUNT_VIEW_CONTAINER);
+		// Check if view is registered, if it isn't, register the view
+		if (!Registry.as<IViewsRegistry>(ViewContainerExtensions.ViewsRegistry).getView(newProvider.addedProvider.id)) {
+			try {
+				Registry.as<IViewsRegistry>(ViewContainerExtensions.ViewsRegistry).registerViews([{
```

Comment:
there may be a better place to put the view registration, such as one of the .contribution files, that is only loaded once so this gets added deterministically.